src/routes.js

```diff
@@ -1,10 +1,26 @@
 import Home from './pages/Home.vue';
 import Portfolio from './pages/Portfolio.vue';
-import Stocks from './pages/Stocks.vue';
+import Market from './pages/Stocks.vue';
+
+import Config from './config';
+
+const { home, portfolio, market } = Config.routes;
 
 export const routes = [
-    { path: '/', component: Home },
-    { path: '/portfolio', component: Portfolio },
-    { path: '/stocks', component: Stocks },
-    { path: '*', redirect: '/' },
+    {
+        path: home,
+        component: Home
```

Comment:
I just realized, that probably for a better semantic it will be clearer to name all these components as `HomePage`, `PortfolioPage` and `MarketPage`... 🤔